src/_scss/pages/covid19/opportunityProject.scss

```diff
@@ -0,0 +1,71 @@
+@import "all";
+
+.top-page {
+  width: 100%;
+  padding: 0;
+  background-color: $color-gray-lightest;
+}
+
+.content {
+  background-color: $color-white;
+  border: solid thin $color-gray-lighter;
+
+  padding: rem(32) rem(16) rem(40);
+  @include media($medium-screen) {
+    padding: rem(60) rem(40) rem(40);
+  }
+  margin: 0;
+  @include media($medium-screen) {
+    margin: rem(24) auto;
+  }
+}
+
+footer {
```

Comment:
Could these styles all be wrapped in the `top-page` class to make sure they don't have unintended consequences on pages of the site?

Reply:
Probably, but this is the only page that imports that scss, so I don't know how it could bleed over other pages...

Reply:
pushed to sandbox